ui/packages/catalog-ui-search/src/main/webapp/component/result-filter/result-filter.view.js

```diff
@@ -64,7 +63,7 @@ module.exports = Marionette.LayoutView.extend({
     this.handleFilter()
   },
   getFilter() {
-    return this.editorProperties.currentView.transformToCql()
+    return this.editorProperties.currentView.getFilters()
```

Comment:
can we verify that the verification stuff still works? i know it calls this function to show those toast error messages for anyGeo

Reply:
The inline errors are showing, but it doesn't seem to be preventing the save for invalid geos. But it doesn't seem to be happening on 2.19.x either, was it previously preventing the save?

Reply:
by the "save" do you mean when you try to run the search? or something different? we were preventing it there, i don't recall if we were preventing it in the form editors is that's what you mean

Reply:
<img width="657" alt="Screen Shot 2020-08-03 at 9 26 51 AM" src="https://user-images.githubusercontent.com/39923178/89199239-82268180-d56b-11ea-869a-a369a6bebf09.png">
This is for the result-filter component which only has the save or remove options, you can currently save it with an invalid geo

Reply:
got it added